.docs/Get-VSTeamTaskGroup.md

```diff
@@ -0,0 +1,87 @@
+<!-- #include "./common/header.md" -->
+
+# Get-VSTeamTaskGroup
+
+## SYNOPSIS
+
+<!-- #include "./synopsis/Get-VSTeamTaskGroup.md" -->
+
+## SYNTAX
+
+## DESCRIPTION
+
+<!-- #include "./synopsis/Get-VSTeamTaskGroup.md" -->
+
+## EXAMPLES
```

Comment:
Have some examples that don't splat the arguments. I don't mind some but I don't want people to think that is the only way to call them. Also add a description to each example. Point out in the description you are splatting the parameters.